### ISSUE TITLE
Simplify install instructions by changing zip file content

### ISSUE THREAD

**Issue by onemen** [xiaoxiaoflood/firefox-scripts#97]
Hello @xiaoxiaoflood,

Many users are confused by the process of downloading extracting and coping files to two location
 
As a first aid I suggest that you change [fx-folder.zip](https://raw.githubusercontent.com/xiaoxiaoflood/firefox-scripts/master/fx-folder.zip) to firefox-config.zip or similar name and add config.js and default folder directly to a zip file not inside another folder.

For the utils zip I suggest that you zip it already with the chrome folder, then the user can just drag/copy the extracted chrome folder to the profile folder.

Do you have a place where you answer frequently asked questions?
what to answer to users that say:
> The scripts and stuff kinda scare me…

**Comment by 117649** [comment]
> add config.js and default folder directly to a zip file not inside another folder.

Isn't this already the case?
I don't understand.

![](https://user-images.githubusercontent.com/8281345/133964864-c965595f-6223-4937-87f9-2299ccd16ebc.png)

**Comment by 117649** [comment]
> Do you have a place where you answer frequently asked questions?

I don't think anybody ever seen this many users in past several years before TMP is revived. 

> what to answer to users that say:
> 
> > The scripts and stuff kinda scare me…

That's the reason why auto install scripts are exist.

**Comment by xiaoxiaoflood** [comment]
> change fx-folder.zip to firefox-config.zip or similar name

Why do you think it would be better? I don't think so. Maybe `extract-this-to-firefox-folder.zip`...

> add config.js and default folder directly to a zip file not inside another folder

It's already that way.

> For the utils zip I suggest that you zip it already with the chrome folder,

Agreed. I had already thought about that.

> Do you have a place where you answer frequently asked questions?

No. I don't want to be rude to anyone, but I confess that I also don't want to waste time with things such the example you gave, I don't want to convince anyone to do or to use anything. Honestly, everything here I did and do for personal use only. If it's useful to someone else, good and that's it.

By the way, it's possible that one day someone complain with you that it's not working on Nightly or in release or in ESR or in Waterfox... As stated since the first day, I use and target DevEd. I don't code to give or keep compatibility with other versions. I *may* accept pull requests and that's it, nothing more.

**Comment by xiaoxiaoflood** [comment]
Still about `utils.zip`, there's a much larger group of `userChrome.css` users that may be confused seeing the `chrome` folder in the archive, fearing that the extraction would replace the entire folder (deleting their `.css` files). So I'm not entirely convinced that this would be a good change.

**Comment by onemen** [comment]
> fearing that the extraction would replace the entire folder

if chrome folder already exist in the user folder Windows will merge the content of the folder you copied drag and show you confirmation dialog if files are the same

**Comment by xiaoxiaoflood** [comment]
I know that, but this is not the case for everyone. Someone who considers the current instructions hard to follow may not know this. So I'm tending to keep as is.
